src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig

```diff
@@ -0,0 +1,38 @@
+<div class="btn-group multistore-dropdown js-multistore-dropdown">
```

Comment:
Maybe we could add Twig `block` to allow shop developers to extend these templates?